imports/node-app/core/createApolloServer.js

```diff
@@ -59,7 +64,9 @@ export default function createApolloServer(options = {}) {
     schema,
     subscriptions: {
       path
-    }
+    },
+    instrospection: env.GRAPHQL_INSTROSPECTION_ENABLED,
```

Comment:
`INTROSPECTION` and `introspection` are misspelled here, so have no effect.

Reply:
Oh, sorry about that. Will fix.

Reply:
Fixed.